go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/aquasecurity/trivy-plugin-aqua
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2020-8912   
**Check Name:** aws-sdk-go: In-band key negotiation issue in AWS S3 Crypto SDK for golang   
**Severity:** LOW   
**Fixed Version:**    
**Description:** A vulnerability in the in-band key negotiation exists in the AWS S3 Crypto SDK for GoLang versions prior to V2. An attacker with write access to the targeted bucket can change the encryption algorithm of an object in the bucket, which can then allow them to change AES-GCM to AES-CTR. Using this in combination with a decryption oracle can reveal the authentication key used by AES-GCM as decrypting the GMAC tag leaves the authentication key recoverable as an algebraic equation. It is recommended to update your SDK to V2 or later, and re-encrypt your files.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2020-8912